style.css

```diff
@@ -449,6 +472,119 @@ body {
     margin-top: 20px;
     color: #253858;
   }
+
+  /* contact me section */
+
+  .contact-me {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: -30px;
+    background-image: url("./images/contactMe-bg.svg");
+    background-repeat: no-repeat;
+    background-size: 68%;
+    background-position: right top 40%;
+    background-color: #6070ff;
+    min-height: 100vh;
+    margin-top: -80px;
+    border-radius: 115px 0 0 0;
+    padding-top: 100px;
+  }
+
+  .contact-title {
+    color: white;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin-top: 50px;
+  }
```

Comment:
- [ ] Please make sure that your `contact us` section title looks like the provided [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=OrdN01rgF7wtPtkc-0). Also kindly ☺️ decrease the margin between the title and para to make it look as close to the figam design as possible, as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) of the project

> You don't need to implement any functionality that requires JavaScript.
You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Implement the button interactions (enable, hover, etc.).

_**Figma design**_ 

<img width="226" alt="Screenshot 2023-02-22 at 5 01 09 AM" src="https://user-images.githubusercontent.com/44798044/220486318-ae3c6209-e53f-42d8-9ab7-60d5eb7b7ab5.png">


_**Your design**_

<img width="363" alt="Screenshot 2023-02-22 at 5 01 56 AM" src="https://user-images.githubusercontent.com/44798044/220486340-114d6e18-97a8-4e48-9f9f-8e580271ff8f.png">